importexport/musicxml/importmxmlpass2.h

```diff
@@ -130,20 +130,30 @@ class MusicXMLParserLyric {
       };
 
 //---------------------------------------------------------
-//   Notation
+//   Generic Annotation
+//      Most with text base attributes (font-*, placement)
 //---------------------------------------------------------
 
-class Notation {
+class GenericAnnotation {
 public:
-      Notation(const QString& name) { _name = name; }
+      GenericAnnotation(const QString& name, const QString& parent = "",
+                        const SymId& symId = SymId::noSym) { _name = name; _parent = parent; _symId = symId; }
       void addAttribute(const QStringRef name, const QStringRef value);
       QString attribute(const QString& name) const;
       QString name() const { return _name; }
+      QString parent() const { return _parent; }
+      void setSymId(const SymId& symId) { _symId = symId; }
+      SymId symId() const { return _symId; }
+      void setSubType(const QString& subType) { _subType = subType; }
+      QString subType() const { return _subType; }
       QString print() const;
       void setText(const QString& text) { _text = text; }
       QString text() const { return _text; }
 private:
       QString _name;
+      QString _parent { QString("") };
```

Comment:
@lvinken , you meant to remove this one as well, right?

Reply:
yes